<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>791</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainPage{background-color: rgb(255, 255, 255);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <spacer name="verticalSpacer_Top">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>56</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_Left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_Updatecard">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>533</width>
             <height>243</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>533</width>
             <height>243</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_NewCard">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>533</width>
             <height>243</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>533</width>
             <height>243</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_RegisterLost">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>262</width>
             <height>243</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>262</width>
             <height>243</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ChangePWD">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>262</width>
             <height>243</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>262</width>
             <height>243</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_Right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_Bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>18</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>178</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_QR1">
         <property name="minimumSize">
          <size>
           <width>143</width>
           <height>143</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>143</width>
           <height>143</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Image/qr_wechat1.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label_QR1_Service">
           <property name="styleSheet">
            <string notr="true">font-family: &quot;思源黑体 CN Medium&quot;;
font-size: 25px;
font-weight: bold;
line-height: 49px;
letter-spacing: 1px;
color: #414141;
</string>
           </property>
           <property name="text">
            <string>关注驻马店人社公众号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_QR1_Service_Text1">
           <property name="styleSheet">
            <string notr="true">font-family: &quot;思源黑体 CN Medium&quot;;
font-size: 25px;
font-weight: bold;
line-height: 49px;
letter-spacing: 1px;
color: #414141;</string>
           </property>
           <property name="text">
            <string>提供查询社保服务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_QR1_Service_Text2">
           <property name="styleSheet">
            <string notr="true">font-family: &quot;思源黑体 CN Medium&quot;;
font-size: 25px;
font-weight: bold;
line-height: 49px;
letter-spacing: 1px;
color: #414141;</string>
           </property>
           <property name="text">
            <string>提供缴费通知服务</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_QR2">
         <property name="minimumSize">
          <size>
           <width>143</width>
           <height>143</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>143</width>
           <height>143</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Image/qr_wechat2.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_QR2_Service">
             <property name="styleSheet">
              <string notr="true">font-family: &quot;思源黑体 CN Medium&quot;;
font-size: 25px;
font-weight: bold;
line-height: 49px;
letter-spacing: 1px;
color: #414141;
</string>
             </property>
             <property name="text">
              <string>关注驻马店人社公众号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_QR2_Service_Text1">
             <property name="styleSheet">
              <string notr="true">font-family: &quot;思源黑体 CN Medium&quot;;
font-size: 25px;
font-weight: bold;
line-height: 49px;
letter-spacing: 1px;
color: #414141;
</string>
             </property>
             <property name="text">
              <string>提供查询社保服务</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_QR2_Service_Text2">
             <property name="styleSheet">
              <string notr="true">font-family: &quot;思源黑体 CN Medium&quot;;
font-size: 25px;
font-weight: bold;
line-height: 49px;
letter-spacing: 1px;
color: #414141;
</string>
             </property>
             <property name="text">
              <string>提供缴费通知服务</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
